spec/features/viewing_chitters_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'pg'
+require 'orderly'
+require 'spec_helper'
+
+feature 'posting to chitter' do
+  scenario 'user can post a peep to chitter' do
+    visit '/'
+    fill_in 'peep', with: 'Loving life #brunch' 
+    click_button 'Peep'
```

Comment:
This could be made into a spec helper function, same for relating actions in other tests